src/Subscription.php

```diff
@@ -704,7 +704,7 @@ public function paddleEmail()
      */
     public function paymentMethod()
     {
-        return (string) $this->paddleInfo()['payment_information']['payment_method'];
+        return (string) ($this->paddleInfo()['payment_information']['payment_method'] ?? '');
```

Comment:
Maybe better to return null here, and update the phpdoc. Seems odd to pretend we have a payment method when we don't?